docs/articles/documentation/guides/advanced-guides/multiple-browser-windows.md

```diff
@@ -162,3 +162,19 @@ test('Close a specific window', async t => {
     await t.closeWindow(window1);
 });
 ```
+
+## Limitations
+
+This section describes the limitations of multiple browser windows testing in TestCafe.
+
+### Limited Video Recording
+
+When you launch TestCafe with [video recording](./screenshots-and-videos.md) enabled, the resulting video scales to the initial (parent) browser window.
+
+When a child window opens, TestCafe continues recording in that window. If the child windows' size differs, the segments of the video that are recorded in these windows are resized to fit the aspect ratio of the main browser window. That part of the recording appears stretched as a result.
```

Comment:
When we say "segment" about an image, we mean the geometrical sense, i.e. a limited area.
When I saw "segments of the video", I thought you also mean "part of the whole recorded area".

I may be wrong, but maybe you'd better check the meaning of "segment" related to videos with Dirk?

Reply:
I agree, this is ambiguous.  Maybe `portion` is better